components/alert/test/alert.visual-diff.js

```diff
@@ -0,0 +1,36 @@
+const puppeteer = require('puppeteer');
+const VisualDiff = require('@brightspace-ui/visual-diff');
+
+describe('d2l-alert', () => {
+
+	const visualDiff = new VisualDiff('alert', __dirname);
+
+	let browser, page;
+
+	before(async() => {
+		browser = await puppeteer.launch();
+		page = await visualDiff.createPage(browser, { viewport: { width: 800, height: 1200 } });
+		await page.goto(`${visualDiff.getBaseUrl()}/components/alert/test/alert.visual-diff.html`, {waitUntil: ['networkidle0', 'load']});
+		await page.bringToFront();
+	});
+
+	after(async() => await browser.close());
+
+	[
+		'default',
+		'success',
+		'critical',
+		'warning',
+		'close',
+		'button',
+		'button-close',
+		'rtl'
```

Comment:
Missing "error" and "call-to-action" here though :)

Reply:
I wasn't sure whether they were worth adding more test cases for since they aren't documented as part of `design.d2l` or the README. They're really only there for backwards compatibility since they share colors with `critical` and `default`.

Reply:
I think it's worth it solely for the future-case when we go into this code and are like "why is it listing `error` AND `critical` in the CSS? Must be a mistake -- DELETE DELETE DELETE" and this would catch the regression.